documentation/remote-host-object.md

```diff
@@ -0,0 +1,27 @@
+# Support for remote host object
```

Comment:
Can you move this to `documentation/specs/`? I think it's more appropriate there.